src/ert/config/queue_config.py

```diff
@@ -449,11 +415,10 @@ def _validate_queue_driver_settings(
 
 @no_type_check
```

Comment:
Is it a lot of work to get rid of the `@no_type_check` decorator?

Reply:
I think so (and it seems so..) :(